admission/Makefile

```diff
@@ -13,9 +13,11 @@ GOARCH := $(shell go env GOARCH)
 GOFLAGS = -mod=vendor
 export GOFLAGS
 
+.PHONY: all
 all: build
 
 # Run tests
+.PHONY: test
 test: crds
 	test -z "$$(gofmt -s -l . | grep -v '^vendor' | tee /dev/stderr)"
```

Comment:
Can this block be replaced with `$(MAKE) -f $(COMMON_MAKEFILE ) test`??

Reply:
Thank you. I removed `Makefile.common` because it would be needlessly complicated.